pyproject.toml

```diff
@@ -17,11 +17,13 @@ classifiers = [
 
 dependencies = [
     "cached_path>=1.6.2",
-    "ase>=3.22.1",
-    "numpy<2.0.0",
+    "ase>=3.23.0",
+    "numpy==1.26.4",
```

Comment:
Is there a reason that numpy needs to be pinned to this specific version?

Reply:
Yeah this is for the augmentation of using `voigt_6_to_full_3x3_stress` without the upgraded version `np.transpose([[s1, s6, s5],
                         [s6, s2, s4],
                         [s5, s4, s3]])` would give an error. But we don't necessarily need to pin to 1.26.4, maybe higher version also works. While originally we do would like to be < 2.0.0.

Reply:
I see, in that case can we specify a bound, like ">1.26.4,<2.0.0"